cypress/e2e/scenario-2.1-basic-service.cy.js

```diff
@@ -193,7 +192,10 @@ if (Cypress.env("edition") === "iso") {
         .within(() => {
           cy.get('[data-label="version"]').trigger("click"); //it's done to avoid flake where the tooltip comes in a way and click ins't triggered
         });
-      cy.get('[data-testid="selected-version"]').should("have.text", "Version: 2");
+      cy.get('[data-testid="selected-version"]', { timeout: 30000 }).should(
```

Comment:
I'm not convinced increasing the timeout will resolve your issue

Reply:
I would like to give it a chance, this part of the code is tied to the continuous query, so I think default timeout of 10s could be not enough. 

2 on 3 passed after 4/4 failures, so I would run it few more times to assert it. 

8th run was good but I did not remove the command to merge unit test, that's why it shows it as failed run.

If result won't be consistent I will give it more thought at monday as I don't have other flow for this scenario in my head at the moment

I won't merge it until you will be back at Tuesday, so we can discuss it if you will be in doubt